src/client/finAuthFetch.ts

```diff
@@ -2,13 +2,40 @@
  * Copyright (c) 2022, J2 Innovations. All Rights Reserved
  */
 
-import { finCsrfFetch } from './finCsrfFetch'
+import { finCsrfFetch, isCsrfError } from './finCsrfFetch'
 import { FetchMethod } from './fetchVal'
-import {
-	AuthenticationError,
-	isAuthenticationError,
-} from '../errors/AuthenticationError'
-import { isHttpError } from '../errors/HttpError'
+
+/**
+ * A general authentication error.
+ */
+export class AuthenticationError extends Error {
```

Comment:
The reason I put this in another file was in the case we needed to throw it in any other part of the system.